fastapi_storages/base.py

```diff
@@ -1,7 +1,11 @@
-from typing import BinaryIO
+from pathlib import Path
+from typing import Any, BinaryIO
 
 
 class BaseStorage:  # pragma: no cover
+    def __init__(self, overwrite_existing_files=True) -> None:
```

Comment:
Let's make this a class-variable like these here: https://github.com/aminalaee/fastapi-storages/blob/main/fastapi_storages/s3.py#L21

Reply:
Do you think it is good idea to keep it in env? Would not it be wise to have it as variable? If I want to have two FileSystemStorage's, I can create one with overwrite another without.